src/components/transactions/Bridge/BridgeConfig.ts

```diff
@@ -112,6 +115,44 @@ const prodConfig: Config[] = [
       },
     ],
   },
+  {
+    sourceChainId: ChainId.base,
+    chainSelector: '15971525489660198786',
+    burnMintTokenPool: '0x98217A06721Ebf727f2C8d9aD7718ec28b7aAe34', // TODO: address book
```

Comment:
this is present on address-book with `GhoBase.GHO_CCIP_TOKEN_POOL` (same for `GHO_TOKEN`)
https://github.com/bgd-labs/aave-address-book/blob/v4.10.0/src/ts/GhoBase.ts
lmk if you'd like the price feeds for gho under the same class, we've not added since those can be fetched dynamically